java/src/main/java/com/powsybl/dataframe/network/adders/GeneratorDataframeAdder.java

```diff
@@ -35,6 +35,7 @@ public class GeneratorDataframeAdder extends AbstractSimpleAdder {
 
     private static final List<SeriesMetadata> METADATA = List.of(
             SeriesMetadata.stringIndex("id"),
+            SeriesMetadata.strings("name"),
```

Comment:
It needs to be added in the adder

Reply:
?

Reply:
in createAdder I think a  applyIfPresent(name, row, adder::setName); is missing, or maybe I am wrong

Reply:
it is already in the base class